crates/api/src/local_user.rs

```diff
@@ -125,7 +125,7 @@ impl Perform for GetCaptcha {
       "easy" => gen(Difficulty::Easy),
       "medium" => gen(Difficulty::Medium),
       "hard" => gen(Difficulty::Hard),
-      _ => gen(Difficulty::Medium),
+      _ => gen(Difficulty::Easy),
```

Comment:
You could change this to Difficulty::default(), then theres one less place to worry about.

Reply:
That doesn't exist there, these enums are from the captcha crate (Only easy, medium, and hard). So `Difficulty::Easy` is correct here.